ViAn/GUI/mainwindow.cpp

```diff
@@ -217,28 +235,62 @@ void MainWindow::resizeEvent(QResizeEvent* event) {
 }
 
 /**
+ * @brief MainWindow::on_slider_moving
+ * Sends frame numbers to the video player whilst the slider is moving
+ */
+void MainWindow::on_slider_moving(){
+    QPoint local_mouse_pos = ui->videoSlider->mapFromGlobal(QCursor::pos());
+    float pos_ratio = local_mouse_pos.x() / (float )ui->videoSlider->size().width();
+    int slider_range = ui->videoSlider->maximum() - ui->videoSlider->minimum();
+    int slider_pos_under_mouse = ui->videoSlider->minimum() + slider_range * pos_ratio;
+
```

Comment:
This code is very similar to code in `on_slider_click`. Maybe you could extract it t oa separate method, or call `on_slider_click` from `on_slider_moving`.